sdks/sdk_features_matrix.md

```diff
@@ -0,0 +1,21 @@
+= AeroGear SDK Feature Matrix
```

Comment:
Is the proposal the right place for it? Feels like this should be in the document repo.